Documentation/ceph-object-bucket-claim.md

```diff
@@ -102,18 +102,92 @@ metadata:
   labels:
     aws-s3/object [1]
 provisioner: rook-ceph.ceph.rook.io/bucket [2]
-parameters: [3]
+parameters:
   objectStoreName: my-store
   objectStoreNamespace: rook-ceph
-  region: us-west-1
+  region: us-west-1 [3]
   bucketName: ceph-bucket [4]
 reclaimPolicy: Delete [5]
 ```
 1. `label`(optional) here associates this `StorageClass` to a specific provisioner.
 1. `provisioner` responsible for handling `OBCs` referencing this `StorageClass`.
-1. **all** `parameter` required.
+1. `region`(optional) defines the region in which the bucket should be created. For RGW, this maps to [zonegroup](https://docs.ceph.com/en/latest/radosgw/multisite/#zone-groups) of the server. If user does not define `region`, Rook will fill this value accordingly. For examples please check at the end of the doc.
 1. `bucketName` is required for access to existing buckets but is omitted when provisioning new buckets.
 Unlike greenfield provisioning, the brownfield bucket name appears in the `StorageClass`, not the `OBC`.
 1. rook-ceph provisioner decides how to treat the `reclaimPolicy` when an `OBC` is deleted for the bucket. See explanation as [specified in Kubernetes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retain)
 + _Delete_ = physically delete the bucket.
 + _Retain_ = do not physically delete the bucket.
+
+
+##### Storage Class for simple local object store
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: rook-ceph-delete-bucket
+provisioner: rook-ceph.ceph.rook.io/bucket
+reclaimPolicy: Delete
+parameters:
+   objectStoreName: my-store
+   objectStoreNamespace: rook-ceph
+```
+
+##### CRDs for Storage Class, if region defined
```

Comment:
A brief description of the example would also help here. You might also consider cutting out sections of the yaml that is not needed for the example, just to show the properties that must be set for a custom region.
```suggestion
##### Example: Custom region
```

Reply:
@travisn : I am not sure what part can be cut off, for the region, a `Zonegroup` need to define. In the RGW use cases most like to have zone and zone group coexists. Rook mandates realm as part of `ZoneGroup` CRD. So in short with the current implementation users need to define all three.